### ISSUE TITLE
Yae Fischl and Lisa do not target cores

### ISSUE THREAD

**Issue by RarePossum** [genshinsim/gcsim#1342]
Yae E, Oz and Lisa Q never target cores, Even if they target at a lower priority, currently they will never hit a core.

Seperate issue is bloom hitbox size but thats a big question mark.

**Comment by k0l11** [comment]
Fischl is a wontfix from my side, since faithfully recreating the targeting score calculation is a huge pain in the ass and not worth it imo (vast majority of the time Oz will be targeting the closest enemy anyways).
Because of that, I've set this issue to close once #1373 is merged.
